app/controllers/judges_controller.rb

```diff
@@ -157,47 +156,62 @@ def activity
     @year = params[:year] || Contest.order(:start).last.start.year
 
     # Create a hash of Category.id => Category.name
+    # TODO should this be category and aircat instead of name?
+    #   If a flight was only recorded in a synthetic
+    #   category, that will be the category found
     ch = Category.pluck(:id, :name).to_h
```

Comment:
@djmolny One idea would be to synthesize a name from the category and aircat, e.g. anything with `category: 'advanced', aircat: 'P'` would come out as "Advanced Power".